week3/src/components/main/CuisineSelection.jsx

```diff
@@ -0,0 +1,39 @@
+// CuisineSelection.jsx
+import React from "react";
+import { CommonBtn, StepBtn } from "../button/Button";
+
+export default function CuisineSelection({
+  onSelectCuisine,
+  selectedCuisine,
+  goBack,
+  goForward,
+}) {
+  return (
+    <div>
+      <CommonBtn
+        onClick={() => onSelectCuisine("KOREA")}
+        selected={selectedCuisine === "KOREA"}
```

Comment:
요거 실행은 잘 될텐데 브라우저 상에서 콘솔창 띄웠을 때 에러 따로 안나나용!?
props로 넘겨받은 걸 활용해서 스타일링할 때는 보통 앞에 `$` 표시를 붙여줘야 에러가 발생하지 않습니당!
```suggestion
        $selected={selectedCuisine === "KOREA"}
```

Reply:
오잉 에러 안뜨던데 오케이 수정할게 ㅋㅋ

Reply:
[Transient props](https://styled-components.com/docs/api#transient-props)
라고 하는데! 앞에 `$`를 붙이면 단순히 스타일링을 위해서 내려주는 props임을 구분해서, 실제 HTML에는 영향이 가지 않도록 해주는 역할이야 :)